src/types.ts

```diff
@@ -428,6 +466,7 @@ export type ExcalidrawImperativeAPI = {
   readyPromise: ResolvablePromise<ExcalidrawImperativeAPI>;
   ready: true;
   id: string;
+  setCustomType: InstanceType<typeof App>["setCustomType"];
```

Comment:
this should be unnecessary after we add the `renderButton` into `customElementsConfig`, and the editor should just handle this ourselves.